### PR TITLE
Update the doc on overrides

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Updated the document on overriding sub-charts as the example was deprecated.

**Related issue(s)**
See also #6600

### CODE REVIEW

File: docs/kyma/05-03-overrides.md [hunk 1]
```diff
@@ -101,44 +101,45 @@ When a sub-chart contains the `values.yaml` file, the information about the char
...
+    appTokenExpirationMinutes: 5
 ```
 
-To override this value, and change it from `60` to `90`, do the following:
+To override this value and change it from `5` to `10`, do the following:
 
 - Create a ConfigMap in the `kyma-installer` Namespace and label it.
-- Add the `application-connector.connector-service.deployment.args.tokenExpirationMinutes: 90` entry to the ConfigMap.
+- Name it after the main component chart in the `resources` folder and add the `-overrides` suffix to it. In this example, that would be `application-connector-overrides`.
+- Add the `connector-service.deployment.args.appTokenExpirationMinutes: 10` entry to the ConfigMap.
```
Comment (magicmatatjahu) [inline docs/kyma/05-03-overrides.md]: Add the `connector-service.deployment.args.appTokenExpirationMinutes: 10` entry to the ConfigMap **data** field

File: docs/kyma/05-03-overrides.md [hunk 2]
```diff
@@ -101,44 +101,45 @@ When a sub-chart contains the `values.yaml` file, the information about the char
...
 All template values for a sub-chart must be prefixed with a sub-chart "path" that is relative to the top-level "parent" chart.
 
-This is not an Kyma Operator-specific requirement. The same considerations apply when you provide overrides manually using the `helm` command-line tool.
+This is not a Kyma Operator-specific requirement. The same considerations apply when you provide overrides manually using the `helm` command-line tool.
 
-Here is an example.
-There's a `core` top-level chart that the Kyma Installer installs.
-There's an `application-connector` sub-chart in `core` with a nested `connector-service` sub-chart.
-In one of its templates, there's a following fragment:
+For example, there's the `connector-service` sub-chart nested in the `application-connector` chart installed by default as part of the [Kyma package](`connector-service` sub-chart).
```
Comment (mmitoraj) [inline docs/kyma/05-03-overrides.md]: The link does not work.

File: docs/kyma/05-03-overrides.md [hunk 3]
```diff
@@ -101,44 +101,45 @@ When a sub-chart contains the `values.yaml` file, the information about the char
...
 deployment:
+  ...
   args:
-    tokenExpirationMinutes: 60
+    ...
+    appTokenExpirationMinutes: 5
 ```
 
-To override this value, and change it from `60` to `90`, do the following:
+To override this value and change it from `5` to `10`, do the following:
```
Comment (mmitoraj) [inline docs/kyma/05-03-overrides.md]: What do you think about making an ordered list for the below steps?

File: docs/kyma/05-03-overrides.md (hunk 2 above)
Comment (kazydek) [reply docs/kyma/05-03-overrides.md]: Thx, good catch ;)

File: docs/kyma/05-03-overrides.md (hunk 3 above)
Comment (kazydek) [reply docs/kyma/05-03-overrides.md]: I say yes :)

**Pull request merged**